pkg/controller/pxc/controller.go

```diff
@@ -1106,14 +1110,23 @@ func (r *ReconcilePerconaXtraDBCluster) createOrUpdate(obj runtime.Object) error
 		case *corev1.Service:
 			object.Spec.ClusterIP = oldObject.(*corev1.Service).Spec.ClusterIP
 		}
-		// log.Log.Info("UPDATING OBJECT", "NAME", metaAccessor.GetObjectMeta().GetName(), "KIND", obj.DeepCopyObject().GetObjectKind().GroupVersionKind().Kind)
+		log.Log.Info("UPDATING OBJECT", "NAME", metaAccessor.GetObjectMeta().GetName(), "KIND", obj.DeepCopyObject().GetObjectKind().GroupVersionKind().Kind)
```

Comment:
should we really print it? at least it should be "Debug" level

Reply:
Will be removed after confirming the way